packages/ace-cli/package.json

```diff
@@ -18,6 +18,7 @@
   "main": "lib/index.js",
   "bin": "bin/ace.js",
   "dependencies": {
```

Comment:
Shouldn't the version of this `ace-cli` package be updated too?

Reply:
indeed!

Reply:
Done https://github.com/daisy/ace/pull/226/commits/34936f751ec56a16fc2ef925f682f191c7d390e1